README.md

```diff
@@ -22,6 +22,16 @@ typings search tape
 # Find an available definition (by name).
 typings search --name react
 
+# Install ambient typings (default from "DT", configurable through `defaultAmbientSource` in `.typingsrc`)
+typings install mocha --ambient --save
+
+# Install non-ambient typings (default from "npm" registry, configurable through `defaultSource` in `.typingsrc`)
+typings install chai --save
+
+# Install typings from particular registry
+typings install env!atom --ambient --save
+typings install npm!bluebird --save
+
 # Install typings (DT is "ambient", make sure to enable the flag and persist the selection in `typings.json`).
 typings install react --ambient --save
```

Comment:
I'd remove this and use your example. I think this is legacy back when it worked with prompts anyway.